packages/jack.rb

```diff
@@ -32,16 +33,16 @@ def self.patch
   end
 
   def self.build
-    system './waf',
-           'configure',
-           '--dbus',
-           '--classic',
-           '--db=yes',
-           '--alsa=yes',
-           '--sndfile=yes',
-           '--autostart=none',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    system "env CFLAGS=-fno-stack-protector \
+      ./waf configure \
+      --dbus \
+      --classic \
+      --db=yes \
+      --alsa=yes \
+      --sndfile=yes \
+      --autostart=none \
+      --prefix=#{CREW_PREFIX} \
+      --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
I assume you already tried `#{CREW_OPTIONS}`?_

Reply:
Usually it's the target line in CREW_OPTIONS that manages to gum things up.